stories/molecules/gv-table.stories.js

```diff
@@ -198,11 +197,10 @@ export const dynamicRows = makeStory(conf, {
           {
             field: 'name',
             label: 'Name',
-            type: (item) => (item._new ? 'gv-input' : item._select ? 'gv-select' : 'label'),
+            type: (item) => (item._new ? 'gv-input' : 'label'),
```

Comment:
what about keeping the story even if it's not working well right now ? This way when we'll work on a fix we already have the story.